externs/shaka/manifest.js

```diff
@@ -237,6 +237,33 @@ shaka.extern.Variant;
 shaka.extern.CreateSegmentIndexFunction;
 
 
+/**
+ * @typedef {{
+ *   method: string,
+ *   cryptoKey: !webCrypto.CryptoKey,
+ *   iv: (!Uint8Array|undefined),
+ *   firstMediaSequenceNumber: number
+ * }}
+ *
+ * @description
+ * AES-128 key and iv info from the HLS manifest.
+ *
+ * @property {string} method
+ *   The key method defined in the HLS manifest.
+ * @property {!webCrypto.CryptoKey} cryptoKey
+ *   Web crypto key object of the AES-128 CBC key.
+ * @property {(!Uint8Array|undefined)} iv
+ *   The IV in the HLS manifest if defined. See HLS RFC 8216 Section 5.2 for
+ *   handling undefined IV.
+ * @property {number} firstMediaSequenceNumber
+ *   The starting Media Sequence Number of the playlist, used when IV is
+ *   undefined.
+ *
+ * @exportDoc
+ */
+shaka.extern.HlsAes128Key;
```

Comment:
I recommended referencing HLS here because AES-128 is otherwise just the name of a block cipher.  It's the underlying block cipher in two common, mutually-incompatible DRM schemes (cenc, cbcs), two less common, equally incompatible DRM schemes (cens, cbc1), and in yet another incompatible scheme used in HLS (this one).  The differences are in the block cipher modes and in which parts of the segment get encrypted.